app/controllers/account/admin/users_controller.rb

```diff
@@ -7,7 +7,13 @@ class UsersController < ApplicationController
       before_action :admin?, only: [:change_role]
 
       def index
-        @users = User.page(params[:page]).per(20)
+        @users_scope = User.all
```

Comment:
I thing it is a bad idea to get all users. What will be if we have a lot of users? Somebody mentioned about it on last Demo. Maybe somebody from mentors can say something about  it?